packages/mobile/src/components/Button/Button.tsx

```diff
@@ -26,6 +27,7 @@ export const CustomButton = ({
   ...buttonProps
 }: CustomButtonProps) => {
   const isPrimary = variant === 'primary' ? true : false;
+  const isSolid = variant === 'primary__solid' ? true : false;
```

Comment:
we can simplify it to variant === 'primary__solid'

Reply:
Yep! Thanks!